sunset/code/modules/map_fluff/maps.dm

```diff
@@ -0,0 +1,13 @@
+
+GLOBAL_DATUM_INIT(using_map, /datum/map/sunset, new) // too lazy to make a dynamic system for this
```

Comment:
Just load it via JSON info to the map datum instead of a hacky system

Reply:
hm good idea

Reply:
Done